src/commands/junit/upload.ts

```diff
@@ -76,6 +89,14 @@ export class UploadJUnitXMLCommand extends Command {
     `,
     examples: [
       ['Upload all jUnit XML test report files in current directory', 'datadog-ci junit upload --service my-service .'],
+      [
+        'Discover and upload all jUnit XML test report files doing recursive search in current directory',
+        'datadog-ci junit upload --service my-service --auto-discovery .',
+      ],
+      [
+        'Discover and upload all jUnit XML test report files doing recursive search in current directory, ignoring ./src/ignored-module-a and ./src/ignored-module-b',
+        'datadog-ci junit upload --service my-service --ignored-paths ./src/ignored-module-a,./src/ignored-module-b --auto-discovery .',
```

Comment:
do we say somewhere that `--ignored-paths` are comma separated?

Reply:
yes, it is said in the readme